mithril-aggregator/src/event_store/transmitter_service.rs

```diff
@@ -36,31 +35,10 @@ where
 }
 
 impl TransmitterService<EventMessage> {
-    /// Craft and send an [EventMessage] given the serializable data.
-    /// This method is done in a way to make as simple as possible to send a
-    /// message and make any error not to cause a business failure. A warning is
+    /// Send an [EventMessage].
+    /// This method make any error not to cause a business failure. A warning is
```

Comment:
Can you make this message a bit more clear?

Reply:
Is this any clearer ? 
```
Send an [EventMessage].
An error when sending a message has no impact on the business. 
If there is one, a warning is issued so the resulting error may be ignored by the caller.
```